docs/documentation/guides/task-library.mdx

```diff
@@ -184,6 +184,28 @@ We support the following log levels:
 
 [reference docs](/sdk/io/logger)
 
+## `store`
+
+The store object exposes several namespaced **Key-Value Stores** you can access inside of your Jobs. This is useful for storing small amounts of serializable data for later retrieval:
+
+```ts
+// store some data
+await io.store.job.set("💾", "disk-A", "Doom 1.2 Demo #3");
+
+// get it again later
+const value = await io.store.job.get<string>("💾", "disk-A");
+```
+
+If you want to acess the store from outside a run (e.g. just from your backend) you should use [client.store](/sdk/triggerclient/store) instead.
+
+The following namespaces are at your disposal:
+
+- `store.env` to access and store data within the **Environment**
```

Comment:
I'd explain how this allows you to share data across jobs and `store.job` allows you to share data across runs